### DISCUSSION TITLE
Extract path data

### DISCUSSION THREAD

**Discussion by Mtehabsim** [stella-cv/stella_vslam#585] [Q&A]
hello, running the localization on a 360 video, and generated a map.msg and I would like to extract the path it was extracted as lets say first position 0,0 and how the video moved around.

Also i am not familiar with concepts like landmark and keyframes. If you can provide resources to learn more about them and how generally the algorithm works. Thanks

**Comment by Skwangles** [comment]
- keyframes are just the saved frames - i.e. so we aren't having to save every single frame but can still represent a graph of the camera's movements.

- landmarks refer to recognisable points by the system, in stella vslam's case using ORB descriptors to identify and save recognisable points (usually high contrast edges) https://medium.com/@deepanshut041/introduction-to-orb-oriented-fast-and-rotated-brief-4220e8ec40cf.

If you just want the path taken, you can just use the trajectory information output by the example run_video_slam and run_image_slam programs with `--eval-log-dir ./Trajectory` as an argument - This argument "stores the trajectory and tracking times at this path (Specify the directory where it exists.)'
